### ISSUE TITLE
Feature Request: A refresh button on the home screen along with an indication to how much time has passed since the last refresh

### ISSUE THREAD

**Issue by arena** [ProgressiveCoders/Spoke#1359]
**Problem**
You must manually refresh to see new todo items (which is way better than auto-refresh). However, when you are on the home screen it isn't obvious that you need to refresh. Messages might sit needing replies when new Spoke users check their home screen and think they have nothing to reply to.

**Solution**
- Add a refresh button
- Add a time-since-last-refresh indicator

**Mockup**
Note that other parts of the interface may have changed (or not exist yet) but the part to focus on is the upper right hand corner where you can see:
- String: "Last updated about" + approx-time-since-last-refresh + "ago."
- Button with label: "Refresh" + refresh icon
Note: if you want to do this incrementally and just add a refresh button with no time count or icon that would also be helpful. (if you want to do that, we could split this feature request into more granular pieces)

![Group 13](https://user-images.githubusercontent.com/411070/72104256-e1772880-32df-11ea-94f7-93b424fed0e9.png)

**Comment by ibrand** [comment]
We're going to put his on hold because of work in #207 that will make this not needed anymore!

**Comment by arena** [comment]
Since this solution is potentially problematic in terms of performance I am going to simplify it to something with slightly less confusing messaging and no extra features. That way, it can be unblocked and also a good first issue. 

Thoughts:
1. "You have nothing to do" is not true -- you (the texter) need to come back later and check for replies.
2. Adding a refresh button seemed like a good idea but at scale would probably be nightmare. I'm not attached to that button or an expensive query of how many messages were sent.

**Comment by arena** [comment]
actually, we should close this issue and open a new one about a better message to replace "you have nothing to do"

**Comment by arena** [comment]
see issue https://github.com/MoveOnOrg/Spoke/issues/1921
